nrf-hal-common/Cargo.toml

```diff
@@ -52,7 +59,8 @@ version = "0.2.3"
 
 [features]
 doc = []
-default = ["52832"]
+default = ["51"]
```

Comment:
We should probably not have a default feature at all – this crate's purpose is just to be reexported by the facade crates, and there's not really a reason to prefer one chip over another here